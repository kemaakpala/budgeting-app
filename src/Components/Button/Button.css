.btn {
  width: 40px;
  height: 40px;
  background-color: rgba(205, 179, 128, 1);
  box-shadow: 0px 4px rgba(205, 179, 128, 0.69);
  border: none;
  color: #033649;
  cursor: pointer;
}
.btn:hover {
  box-shadow: 0px 4px rgba(205, 179, 128, 0.69);
  background-color: rgba(205, 179, 128, 0.9);
  color: #036564;

}
.btn.rounded {
  border-radius: 50%;
}

.btn.roundedCorners {
  border-radius: 1rem;
}