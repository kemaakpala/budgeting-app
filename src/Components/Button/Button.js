import React from 'react';
import './Button.css';

export default (props) => <button className={`btn ${props.variant} ${props.className}`} title={props.title}>{props.children}</button>