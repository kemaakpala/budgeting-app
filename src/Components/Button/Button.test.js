import React from 'react';
import { render } from '@testing-library/react';
import Button from './Button';
import { faItalic } from '@fortawesome/free-solid-svg-icons';

describe('<Button />', () => {
  it('should render button snapshot', () => {
    const props = {
      title: 'testButton',
      variant: 'testVariant',
      className: 'testClassName'
    }
    const { container: { firstChild }} = render(<Button {...props }>test</Button>)
    expect(firstChild).toMatchSnapshot();
  })
})