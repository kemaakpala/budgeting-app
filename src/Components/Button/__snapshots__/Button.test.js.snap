// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render button snapshot 1`] = `
<button
  class="btn testVariant testClassName"
  title="testButton"
>
  test
</button>
`;
